.github/workflows/build.yaml

```diff
@@ -0,0 +1,45 @@
+name: build
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    env:
+      CGO_ENABLED: "1"
+
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15
+        id: go
+
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Install Libraries
+        run: |
+          sudo apt-get update
+          sudo apt install -y bison build-essential cmake flex git libedit-dev \
+              libllvm8 llvm-8-dev libclang-8-dev libllvm9 llvm-9-dev libclang-9-dev \
+              python zlib1g-dev libelf-dev libtinfo5 libfl-dev
+          git clone https://github.com/iovisor/bcc.git
+          mkdir bcc/build; cd bcc/build
+          cmake ..
+          make
+          sudo make install
+
+      - name: Lint
```

Comment:
Lint should come before the Install Libraries step, no reason to wait for the prerequisites to install before linting

Reply:
Sure! Addressing this in the next commit.